src/main/java/stream/flarebot/flarebot/permissions/PerGuildPermissions.java

```diff
@@ -41,13 +43,15 @@
                 }
             }
         }
-        return hasPerm;
-    }
-
-    public boolean hasPermission(Member user, Permission permission) {
-        Permission.Reply reply = getPermission(user, permission);
-        return reply == Permission.Reply.ALLOW ||
-                (reply == Permission.Reply.NEUTRAL && permission.isDefaultPerm());
+        if (hasPerm == Permission.Reply.NEUTRAL) {
+            if (user.hasPermission(permission.getDiscordPerm()) && permission.getDiscordPerm().size() > 0) {
```

Comment:
Why do you check if it's empty second, also isn't there an isEmpty?

Reply:
Well if it's empty it'll always return true, and I'll check